packages/twenty-front/src/modules/object-record/object-filter-dropdown/components/ConfigurableFilterDropdownContent.tsx

```diff
@@ -0,0 +1,67 @@
+import { FilterDefinition } from '@/object-record/object-filter-dropdown/types/FilterDefinition';
+import { DropdownMenuSeparator } from '@/ui/layout/dropdown/components/DropdownMenuSeparator';
+import { isActorSourceCompositeFilter } from '../utils/isActorSourceCompositeFilter';
+import { ObjectFilterDropdownDateInput } from './ObjectFilterDropdownDateInput';
+import { ObjectFilterDropdownNumberInput } from './ObjectFilterDropdownNumberInput';
+import { ObjectFilterDropdownOptionSelect } from './ObjectFilterDropdownOptionSelect';
+import { ObjectFilterDropdownRatingInput } from './ObjectFilterDropdownRatingInput';
+import { ObjectFilterDropdownRecordSelect } from './ObjectFilterDropdownRecordSelect';
+import { ObjectFilterDropdownSearchInput } from './ObjectFilterDropdownSearchInput';
+import { ObjectFilterDropdownSourceSelect } from './ObjectFilterDropdownSourceSelect';
+import { ObjectFilterDropdownTextSearchInput } from './ObjectFilterDropdownTextSearchInput';
+
+type ConfigurableFilterDropdownContentProps = {
+  filterDefinitionUsedInDropdown: FilterDefinition;
+};
+
+export const ConfigurableFilterDropdownContent = ({
+  filterDefinitionUsedInDropdown,
+}: ConfigurableFilterDropdownContentProps) => {
+  return (
+    <>
+      {[
+        'TEXT',
```

Comment:
Please use a constant and create a boolean variable with a meaningful naming for this predicate